doc/python/decor.rst

```diff
@@ -35,9 +35,10 @@ Cable cell decoration
 
         .. code-block:: Python
 
+            import arbor as A
             # Set cell-wide values for properties for resistivity and capacitance
-            decor = arbor.decor()
-            decor.set_property(rL=100, cm=0.1)
+            decor = A.decor()
+            decor.set_property(rL=100 [unit], cm=0.1[unit])
```

Comment:
Hi,

the units are explained here

https://github.com/arbor-sim/arbor/blob/7d20a79a823c8e3ec6659dbc11154d27691ba171/python/cells.cpp#L626

in the code. Note, however, that only the dimension needs to match, not the scale.
So, you could happily pass `kOhm` to any resistance-type argument and Arbor will
scale it properly.